composer.json

```diff
@@ -8,7 +8,7 @@
         "console",
         "framework",
         "zendframework",
-        "zf2"
+        "zf3"
```

Comment:
I'd argue we should drop the version-specific tags, and just stick with `zendframework`.